code/modules/mob/living/silicon/ai/icons.dm

```diff
@@ -163,14 +165,87 @@ var/list/datum/ai_icon/ai_icons
 /datum/ai_icon/trapped
 	name = "Trapped"
 	alive_icon = "ai-hades"
-	dead_icon = "ai-hades_dead"
+	dead_icon = "ai-hades-crash"
 
-/datum/ai_icon/triumvirate_static
+/datum/ai_icon/triumvirate
 	name = "Triumvirate"
 	alive_icon = "ai-triumvirate"
 	alive_light = "#020B2B"
 
 /datum/ai_icon/triumvirate_static
 	name = "Triumvirate Static"
-	alive_icon = "ai-static"
+	alive_icon = "ai-triumvirate-malf"
 	alive_light = "#020B2B"
+
+//Eros Research Platform Ports
+
+/datum/ai_icon/clown2
+	name = "Honk"
+	alive_icon = "ai-clown2"
+	dead_icon = "ai-clown2-crash"
+
+/datum/ai_icon/boxfort
+	name = "Boxfort"
+	alive_icon = "ai-boxfort"
+	dead_icon = "ai-boxfort_dead"
+
+/datum/ai_icon/ravensdale
+	name = "Ravensdale"
+	alive_icon = "ai-ravensdale"
+	dead_icon = "ai-ravensdale-crash"
+
+/datum/ai_icon/gentoo
+	name = "Gentoo"
+	alive_icon = "ai-gentoo"
+	dead_icon = "ai-gentoo_dead"
+
+/datum/ai_icon/serithi
+	name = "Serithi"

```

Comment:
Don't mind the path being named such, but you may want to have the name not be a distinct CKey. Same for the Ravens.